code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -82,3 +82,6 @@
 
 	/// Boolean for sanity points loss effects
 	var/sanity_lost = FALSE
+
+	/// List of EGO Gifts
+	var/list/ego_gifts/ego_gift_list = list()
```

Comment:
```suggestion
	var/list/ego_gift_list = list()
```
List with pre-defined types will not be supported sooner or later by Byond, from what I've heard.

Reply:
Linters require it to be a pre-defined type list.